experiments/create_feature_generation_training_sets_2026_09_04/tests/test_hydrate.py

```
@@ -0,0 +1,309 @@
+
+        result = hydrate_classifier(
+            labels,
+            records,
+            platform="bluesky",
+            classifier_name="is_political",
+        )
+
+        assert list(result.columns) == ["uri", "label_timestamp", "text", "is_political"]
+        assert len(result) == 1
+        assert result.iloc[0]["text"] == "hello bluesky"
+        assert result.iloc[0]["is_political"] == True
```

Ruff (0.16.3)

[error] Avoid equality comparisons to `True`; use `result.iloc[0]["is_political"]:` for truth checks

Replace with `result.iloc[0]["is_political"]`

(E712)